kokoro/build_binaries_oak_containers.sh

```diff
@@ -18,21 +18,26 @@ cd "$(dirname "$0")/.."
 ./scripts/docker_run nix develop .#containers --command just kokoro_oak_containers
 
 mkdir -p "$KOKORO_ARTIFACTS_DIR/test_logs/"
-cp ./target/nextest/default/*.xml "$KOKORO_ARTIFACTS_DIR/test_logs/" || true
+cp --preserve=timestamps \
```

Comment:
Does the timestamp matter for `test_logs`?

Reply:
No, makes no difference. It is just more consistent. The log file is closed and will continue to show the last time anything was written to it.